backend/app/api/api_v1/endpoints/projects.py

```diff
@@ -87,6 +107,71 @@ async def update_json_data_for_prefix_in_project(
     return JSONDataOut(can_edit=True, data=data, task_id=task_id)
 
 
+@router.post(
+    "/create/",
+    status_code=status.HTTP_201_CREATED,
+    dependencies=[Depends(is_admin_or_superuser), Depends(is_user_active)],
+)
+async def create_new_project(
+    current_user: Annotated[UserBase, Depends(utils.get_current_user)],
+    user_github_id: int,
+    root_path: Path,
+    translation_language: str,
+):
+    try:
+        current_user = get_user(current_user.github_id)
+        source_user = get_user(user_github_id)
+    except ValidationError as e:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=f"User {user_github_id} not found. {e}")
+    if root_path.parts[0] == "root":
+        root_path = settings.WORK_DIR.joinpath(root_path)
+    else:
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail=f"Path '{root_path}' not starting in 'root/' directory.",
+        )
+    if not root_path.exists():
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Root path '{root_path}' not found",
+        )
+    directory_list = ["translation", "comment"]
+
+    new_project_paths = create_new_project_file_names(
+        source_user.username, translation_language, root_path, directory_list
+    )
+    source_root_files_names: list[Path] = list(
+        root_path.glob("*.json") if root_path.is_dir() else root_path.parent.glob("*.json")
+    )
+
+    all_paths_list: list[Path] = []
+    try:
+        for root_file, new_file_path in zip(source_root_files_names, new_project_paths):
+            for directory_type_path in new_file_path:
+                create_project_file(Path(root_file), Path(directory_type_path))
+                all_paths_list.append(directory_type_path)
+    except OverrideException as e:
+        for path in all_paths_list:
+            path.unlink()
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail={"rolling_back": f"{all_paths_list}", "error": str(e)},
+        )
+    search.update_indexes(settings.ES_INDEX, settings.ES_SEGMENTS_INDEX, all_paths_list)
+    result = commit.delay(
+        current_user.model_dump(),
+        [str(path) for path in all_paths_list],
+        f"Creating new project for {source_user.username} in {translation_language.upper()} language",
+    )
+
+    return {
+        "user": source_user.username,
+        "translation_language": translation_language,
+        "new_project_paths": new_project_paths,
```

Comment:
Please remove `/app/checkouts/unpublished/` from each path that the endpoint returns under the `new_project_paths`. I don't think we need to expose that, and the frontend does not rely on it.